### DISCUSSION TITLE
求助：有没有人能解答一下？

### DISCUSSION THREAD

**Discussion by asderfgh** [share121/inter-knot#1050] [其他]
![屏幕截图 2024-08-10 214404](https://github.com/user-attachments/assets/b754090b-eef5-4f3c-88ab-99dc204808f3)
![屏幕截图 2024-08-10 214430](https://github.com/user-attachments/assets/0101f18f-b492-4e46-8bcf-8c05d56d4ea6)
![屏幕截图 2024-08-10 214451](https://github.com/user-attachments/assets/0045650a-cd43-4c8c-bbfa-2bd9d1328459)
我不是很懂电脑.有没有好心人能解答一下？这是什么情况？

**Comment by hanshui2077** [comment]
下油猴了吗

**Comment by lllliufeng** [comment]
没让装扩展？
如何安装

> 您需要先安装一个用户脚本管理器扩展，如 Tampermonkey或 Violentmonkey后才能安装该脚本。

**Comment by shilijia-siweikongjian** [comment]
问个问题，那你是怎么发的帖？

**Reply by hanshui2077** [reply]
可以直接上github发，我苹果手机就这样的

**Comment by xiubachi** [comment]
你没油猴怎么进来的

**Comment by Hanabiiiiii** [comment]
点一下油猴插件，允许油猴插件访问这个网址就行了
